sdk/appconfiguration/app-configuration/package.json

```diff
@@ -128,7 +128,7 @@
     "karma-sourcemap-loader": "^0.3.8",
     "mocha": "^7.1.1",
     "mocha-junit-reporter": "^1.18.0",
-    "nock": "^12.0.3",
+    "nock": "^13.0.0",
```

Comment:
Can you also revert the workaround added for this?
https://github.com/Azure/azure-sdk-for-js/pull/13474

Reply:
There are also tests added in the recorder which would need updates I think.
And test the tables test again after removing the workaround by generating the recording and playing it back.

Reply:
It doesn't seem to exist anymore 😮